# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/base/nsAccUtils.h
# Commit: 987e73f47fda
# Full Hash: 987e73f47fda36f43597b6943f4c9d3509094561
# Author: James Teh <jteh@mozilla.com>
# Date: 2020-05-11 21:47:06
# Regressor Bug: 1572860
# File Overlap Count: 1
# Description:
#   Bug 1572860: Remove the ability for an outer document to override live region markup in an inner iframe document. r=MarcoZ
#   
#   This is something that was implemented a long time ago, but it isn't covered in any spec, other browsers don't implement it and I don't know of any usage in the wild.
#   This doesn't work with OOP iframes, since what we're doing here requires the documents to be in the same process.
#   Given it isn't used or specified, the simplest solution is to just remove the behaviour altogether.
# ==============================================================================

diff -r fabe539cdc5e -r 987e73f47fda accessible/base/nsAccUtils.h
--- a/accessible/base/nsAccUtils.h	Mon May 11 10:33:28 2020 +0000
+++ b/accessible/base/nsAccUtils.h	Mon May 11 11:13:43 2020 +0000
@@ -83,11 +83,9 @@
    *
    * @param aAttributes    where to store the attributes
    * @param aStartContent  node to start from
-   * @param aTopContent    node to end at
    */
   static void SetLiveContainerAttributes(nsIPersistentProperties* aAttributes,
-                                         nsIContent* aStartContent,
-                                         mozilla::dom::Element* aTopEl);
+                                         nsIContent* aStartContent);
 
   /**
    * Any ARIA property of type boolean or NMTOKEN is undefined if the ARIA